packages/execution-environments/src/iframe/IFrameSnapExecutor.ts

```diff
@@ -19,6 +19,7 @@ export class IFrameSnapExecutor extends BaseSnapExecutor {
       name: 'child',
       target: 'parent',
       targetWindow: self.parent,
+      targetOrigin: '*',
```

Comment:
What's the reason for this change?

Reply:
This was the default before, but was changed in the latest release: https://github.com/MetaMask/post-message-stream/commit/75ac1a0c7c21a09d194dd857d06ab2a1157caafd